src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheck.java

```diff
@@ -572,110 +637,65 @@ private boolean hasAnyImportInCurrentGroup(String currentGroup) {
      * @return import valid group.
      */
     private String getImportGroup(boolean isStatic, String importPath) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The method getImportGroup() has an NPath complexity of 120](https://www.codacy.com/app/checkstyle/checkstyle/file/1008138044/issues/source?bid=2193633&fileBranchId=2722517#l639)